### ISSUE TITLE
Allow for more than one 2FA option to be used so there is a back up

### ISSUE THREAD

**Issue by davidclintontitanwebtech** [umbraco/Umbraco.Cloud.Issues#733]
### Issue description

Given the current issues with email verification codes not being sent we have essentially been locked out of some of our cloud accounts as we cannot pass the 2FA verification check.

Ideally users should be able to select more than one 2FA option (i.e email and SMS/auth app) so there is a back up option available in case an issue like the above happens again in future.

**Comment by sergiutesu** [comment]
Referring to this one: https://status.umbraco.io/incidents/scr3zlfswkjf

**Comment by c9mb** [comment]
Yes, I have been having issues with my phone provider not sending SMS messages in a timely manner - up to a 15min delay, or not at all - and then of course when you re-try, what you receive is inevitably now wrong because it's the previous. I know I probably should use Authenticator as my primary, but authenticator always feels like a hassle to use, so I prefer SMS, but it would be nice to be able to configure both and switch to Authenticator if SMS is troublesome for some reason, rather than remain locked out.

**Comment by c9mb** [comment]
Hah! - well that was prescient - my phone has a "No Service" status, so SMS is not working for me at the moment.
